docs/howtoguides/enable_realtime_kernel.rst

```diff
@@ -1,92 +1,147 @@
+         sudo pro enable realtime-kernel --variant=intel-iotg
+
+You will need to acknowledge a warning, then you will see a confirmation
+message like the following that the Real-time Ubuntu package has been
+installed:
 
 .. code-block:: text
 
-    One moment, checking your subscription first
-    Updating package lists
-    Skipping installing packages: ubuntu-realtime
-    Real-time kernel access enabled
+   One moment, checking your subscription first
+   Real-time kernel cannot be enabled with Livepatch.
+   Disable Livepatch and proceed to enable Real-time kernel? (y/N) y
+   Disabling incompatible service: Livepatch
+   The Real-time kernel is an Ubuntu kernel with PREEMPT_RT patches integrated.
 
-To install the kernel you can then run:
+   This will change your kernel. To revert to your original kernel, you will need
+   to make the change manually.
 
-.. code-block:: bash
+   Do you want to continue? [ default = Yes ]: (Y/n) Y
+   Updating Real-time kernel package lists
+   Updating standard Ubuntu package lists
+   Installing Real-time kernel packages
+   Real-time kernel enabled
+   A reboot is required to complete install.
+
+You will now need to reboot your machine to complete the process. After
+rebooting, you will be running Real-time Ubuntu.
+
+Install and enable manually
+===========================
+
+To access the Real-time Ubuntu repository but not install the package
+immediately, you can use the ``--access-only`` flag, which was introduced in
+Pro Client version 27.11:
+
+.. code-block:: shell-session
+
+   $ sudo pro enable realtime-kernel --access-only
+
+   One moment, checking your subscription first
+   Real-time kernel cannot be enabled with Livepatch.
+   Disable Livepatch and proceed to enable Real-time kernel? (y/N) y
+   Disabling incompatible service: Livepatch
+   Updating Real-time kernel package lists
+   Skipping installing packages: ubuntu-realtime
+   Real-time kernel access enabled
+
+
+Now that Real-time Ubuntu is accessible, you can install and enable it whenever
+you wish. For example, to install the generic Real-time kernel (not suitable
+for Raspberry Pi):
+
+.. code-block:: shell
 
-    $ sudo apt install ubuntu-realtime
+   sudo apt install ubuntu-realtime
 
-You'll need to reboot after installing to boot into the real-time kernel.
+After rebooting, you'll be running Real-time Ubuntu.
 
-Notes
-=====
+Next steps
+==========
 
-* Real-time kernel is not compatible with Livepatch. If you wish to use the
-  real-time kernel but Livepatch is enabled, ``pro`` will warn you and offer
-  to disable Livepatch first.
+For more information about Real-time Ubuntu and how it can help you, refer
+to the `official Real-time Ubuntu`_ documentation.
 
 .. LINKS
 
 .. include:: ../links.txt
 
-.. _contact our real-time team: https://ubuntu.com/kernel/real-time/contact-us
+.. _kernel_variant: https://canonical-real-time-ubuntu-documentation.readthedocs-hosted.com/en/latest/reference/releases/
```

Comment:
This page is not just about kernel variants.
```suggestion
.. _rt_releases: https://canonical-real-time-ubuntu-documentation.readthedocs-hosted.com/en/latest/reference/releases/
```
... to be updated in citations as well.

Reply:
Thanks for the clarification, and your suggestions. I've applied them now.